rust-runtime/aws-smithy-client/src/builder.rs

```diff
@@ -78,9 +85,8 @@ impl<M, R> Builder<(), M, R> {
     pub fn connector<C>(self, connector: C) -> Builder<C, M, R> {
         Builder {
             connector,
-            standard_retry_config: self.standard_retry_config,
-            retry_policy: self.retry_policy,
             middleware: self.middleware,
+            retry_config: self.retry_config,
```

Comment:
❤️ ty John

Reply:
Ended up needing to refactor this even more. It's now completely decomposed so that it's not possible to set something on the builder that another part of the builder makes use of. For example, previously the builder took both a `timeout_config` and a `connector`, but the connector needs pieces from the `timeout_config`, and those same parts used by the connector are not used where the `timeout_config` is actually used. This made it really easy to end up in a difficult to debug misconfiguration.

It should at least be easier to use correctly now, but a bit less convenient to use.